wherehows-web/app/templates/index.hbs

```diff
@@ -1,15 +1,21 @@
 <div class="row">
-  {{#each model as |featureCard|}}
-    <div class="col-sm-3">
-      <article class="feature-card">
-        {{#link-to featureCard.route}}
-          <span class="feature-card-icon" title={{featureCard.title}}></span>
-          <h3 class="feature-card__title">{{featureCard.title}}</h3>
-          <div class="feature-card__description">
-            {{featureCard.description}}
+  <div class="container">
+    <div class="row">
+      {{#each model as |featureCard|}}
+        <div class="col-md-3">
+          <div class="feature-container">
+            <article class="feature-card">
+              {{#link-to featureCard.route}}
+                <span class="feature-card-icon" title={{featureCard.title}}></span>
```

Comment:
Haha, yea, not used. will delete. And deleted code is always the best
deletedCode===debuggedCode